managers/mapdb/src/main/kotlin/com/justai/jaicf/context/manager/mapdb/BotContextModel.kt

```diff
@@ -1,15 +1,24 @@
 package com.justai.jaicf.context.manager.mapdb
 
 import com.justai.jaicf.context.BotContext
+import com.justai.jaicf.context.DialogContext
 import java.io.Serializable
 
-class BotContextModel(botContext: BotContext): Serializable {
-    val dialogContext = botContext.dialogContext
-    val result: Any? = botContext.result
-    val client = botContext.client.toMap()
-    val session = botContext.session.toMap()
+internal class BotContextModel private constructor(): Serializable {
```

Comment:
Did you check this change with the old MapDB manager? As far as old MapDB works with java.io.Serializable we should double-check all changes in serializable classes

Reply:
Also, we shouldn't change the visibility of public classes.
And anyway, I'd suggest remaining all old code unchanged and mark it deprecated.
And implement jackson-mapdb as completely new manager